app/views/checkout/new.html.erb

```diff
@@ -0,0 +1,7 @@
+<h2>Welcome to checkout</h2>
+<% @cart.shopping_cart_items.each do |cart| %>
+  <div class="images">
+    <%= image_tag cart.item.file.url, style: 'width: 60px; height: auto' %>
+    <%= cart.item.name %>
```

Comment:
Overall, wouldn't it be a good idea to display all items in a table and add a total sum for the order at the bottom. Something like: https://codepen.io/chriscoyier/pen/KpDcm